Examples_Nuget/QuickstartUdf/Program.cs

```diff
@@ -0,0 +1,203 @@
+        {
+            Console.WriteLine($"Running {udfUri} against {arrayList.Arrays.Count} arrays...");
+            var args = RestUtil.SerializeArguments(new Dictionary<string, dynamic>() {{"attr", attribute}});
+            DebugOutput(udfUri, chargedOrg, args);
+
+            // Run against an array with string dimensions
+            var response = TileDB.Cloud.RestUtil.Array.ApplyMultiArray(udfUri, arrayList, arrayNamespace, args);
+            Console.Write($"Result from {udfUri} against {arrayList.Arrays.Count} arrays: ");
+            response.CopyTo(Console.OpenStandardOutput());
+            Console.WriteLine("\n");
+        }
+
+        // Run Array UDF using array as input; `data` arg in the UDF below will contain array slice data
+        // + (Python) UDF declaration for this example: def multi_array_udf(data, attr, scalar)
+        static void UdfMultiArrayExecAsync(string udfUri, RestUtil.ArrayList arrayList, string arrayNamespace,
+            string attribute, string chargedOrg=null)
+        {
+            Console.WriteLine($"Running {udfUri} against {arrayList.Arrays.Count} arrays asynchronously...");
+            var args = RestUtil.SerializeArguments(new Dictionary<string, dynamic>() {{"attr", attribute}});
+            DebugOutput(udfUri, chargedOrg, args);
+
+            // Run against an array with string dimensions
+            var taskResponse = TileDB.Cloud.RestUtil.Array.ApplyMultiArrayAsync(udfUri, arrayList, arrayNamespace, args);
+            Console.WriteLine($"Waiting for result...");
+            Console.Write($"Result from {udfUri} against {arrayList.Arrays.Count} arrays: ");
+            // Accessing the `Result` property of `taskResponse` will block until async task completes
+            taskResponse.Result.Result.CopyTo(Console.OpenStandardOutput());
+            Console.WriteLine("\n");
+        }
+
+        static void UdfUpdate(string uri, string readmeText)
+        {
+            Console.WriteLine($"UDF info before update: {UdfGetInfo(uri).ToJson()}");
+
+            // Update existing UDF tags
+            var curTags = UdfGetInfo(uri).Tags;
+            if (!curTags.Contains("csharp-test"))
+            {
+                curTags.Add("csharp-test");
+            }
+            RestUtil.Udf.UpdateGenericUdf(uri, readmeText, curTags.ToArray());
+
+            Console.WriteLine($"UDF info after update: {UdfGetInfo(uri).ToJson()}\n");
+        }
+
+        static void DebugOutput(string uri, string chargedOrg = null, string args = null)
+        {
+            if (chargedOrg != null)
+            {
+                Console.WriteLine($"Charging {uri} to {chargedOrg} namespace");
+            }
+            if (args != null)
+            {
+                Console.WriteLine($"Passing arguments to UDF: {args}");
+            }
+        }
+
+        static void Run()
+        {
+            RestUtil.Udf.Unshare("tiledb://shaunreed/array-udf", "shaunrd0");
```

Comment:
Please change the URL to variable(s) here and below, so that users only have to edit one place to try it on arrays they can access.